### PR TITLE
Migration/stavetempo

### PR DESCRIPTION
**Pull request by rvilarl**
StaveTempo!

### CODE REVIEW

File: src/stavetempo.ts [hunk 1]
```diff
@@ -4,13 +4,38 @@
...
+import { Stave } from './stave';
+
+export interface StaveTempoOptions {
+  bpm: number;
+  dots: number;
+  duration: string;
+  name: string;
+}
+
+export interface Tempo {
```
Comment (0xfe) [inline src/stavetempo.ts:17]: Both `StaveTempoOptions` and `Tempo` are the same -- let's just use `StaveTempoOptions`.

Comment (rvilarl) [reply src/stavetempo.ts:17]: done

**Comment by rvilarl** [comment]
Ready

**Comment by 0xfe** [comment]
Looks good!

**Pull request merged**